documentation/asciidoc/topics/adding_backup_locations.adoc

```diff
@@ -0,0 +1,66 @@
+[id='adding_backup_locations-{context}']
+= Adding Backup Locations to Caches
+When you configure {brandname} clusters to perform cross-site replication, you
+can add backup locations to your cache configurations.
+
+.Procedure
+
+. Create cache configurations with identical names for each site.
++
+Cache configurations at each site can use different cache modes and backup
+strategies. {brandname} replicates data based on cache names.
++
+. Configure backup locations to go offline automatically with the `take-offline` element.
+.. Do not set a value for the `after-failures` attribute. The default value is `-1` which takes backup locations offline immediately.
```

Comment:
I would remove this.

Reply:
@pruivo Do you think we should remove because it's too strong to say "do not set a value"? Perhaps we could rephrase to 

You should leave the default value of `-1` for the `after-failures` attribute so that backup locations go offline immediately when the remote site becomes unavailable. 

I think the recommendation here is that `-1` means the containers avoid any unnecessary state transfer operations and don't log messages for failed backup attempts. Maybe we should be more explicit about that. @Crumby ?

Reply:
because it is incorrect :) `-1` says to Infinispan to ignore the number of failures when putting the remote site `OFFLINE`. It will decide based on the `min-wait` time. 
And it doesn't affect the state transfer operations. Actually, the state transfer has it own retries configuration and ignores this.

Reply:
thanks @pruivo I think part of the issue here was fuzzy understanding of the observations that Pavel had made in his testing. we were going for some recommendations but the detail might have been a bit garbled. will close this conversation now as I've updated the content.